### PR TITLE
optional LoadBalanced functionality

### PR DESCRIPTION
**Pull request by fitz123**
Check this out

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # defaults file for keepalived
 ---
```
Comment (tersmitten) [inline defaults/main.yml:2]: Why would you set `keepalived_enabled` to `false`?

File: defaults/main.yml [hunk 2]
```diff
@@ -1,5 +1,6 @@
...
+keepalived_enabled: true
```
Comment (fitz123) [inline defaults/main.yml]: Don't really got the point. Don't the role mean to install keepalived?)
I can switch to false if you want. Usually I enable the roles and disable them into individual hosts/group variables

Comment (tersmitten) [inline defaults/main.yml]: I understand, we only enable roles when we want to use them

**Comment by fitz123** [comment]
Looks better?

**Pull request closed without merging**